Failing test `test_simple_transform` under `tests/test_pulse.py`

```
E       AssertionError: assert {'adBlocker':...veloper', ...} == {'adBlocker': ...veloper', ...}
E         Omitting 34 identical items, use -vv to show
E         Differing items:
E         {'consoleErrors': 0} != {'consoleErrors': None}
E         {'trackingProtection': False} != {'trackingProtection': None}
E         Use -v to get the full diff
tests/test_pulse.py:84: AssertionError
```

--- a/tests/test_pulse.py
+++ b/tests/test_pulse.py
@@ -75,10 +75,10 @@ def test_simple_transform(row_to_dict, simple_rdd, spark_context):
         'version': u'1.0.2',
         'requests': requests,
         'disconnectRequests': 1,
-        'consoleErrors': None,
+        'consoleErrors': 0,
         'e10sStatus': 1,
         'e10sProcessCount': 4,
-        'trackingProtection': None
+        'trackingProtection': False,
     }
 
     assert row_to_dict(actual) == expected

